.github/workflows/bot-pr-updated.yml

```diff
@@ -60,3 +60,15 @@ jobs:
         with:
             name: abi
             path: abi.diff
+  check-wiki-changelog:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.author_association == 'COLLABORATOR' || github.event.pull_request.author_association == 'MEMBER' || github.event.pull_request.author_association == 'OWNER'
+    steps:
+    - name: Checkout the latest code (shallow clone)
+      uses: actions/checkout@v2
+      with:
+        ref: develop
+    - name: Check if PR number exists in wiki/Changelog
+      env:
+        GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
+      run: cp --preserve .github/check-wiki-changelog.sh /tmp && /tmp/check-wiki-changelog.sh
```

Comment:
Since the check only uses the API and `develop` and doesn't check out the PR's branch, I think we don't need this level of protection.